### ISSUE TITLE
connection.onopen fire twice each time when new person connect to room

### ISSUE THREAD

**Issue by likev** [muaz-khan/RTCMultiConnection#33]
connection.onopen fire twice for both creator and participator when i use data-only connection.
```javascript
//creator code
var connection = new RTCMultiConnection();
connection.session = {
	data: true
};
var room = 'room1';
connection.open(room);
```

```javascript
//participator code
var connection = new RTCMultiConnection();
connection.session = {
	data: true
};
var room = 'room1';
connection.connect(room);
```

**Comment by muaz-khan** [comment]
Maybe you're using Firefox? Are you using RTCMultiConnection-v2.2.5 or other versions?

It seems that `peer.createDataChannel` along with `ondatachannel` causes multiple data-channel objects until you handle it like following:

1. Offerer invokes `peer.createDataChannel`. Make sure that, this method isn't invoked for answerer (for Firefox only)
2. Answerer sets `peer.ondatachannel` event handler. It works only in Firefox. It'll be not fired in Chrome.

This issue is already fixed in v3.0 which is planned to be released at the end of this week.

**Comment by likev** [comment]
Thanks for reply!
Yes I am using Firefox35, RTCMultiConnection-v2.2.4 on Win7.
RTCMultiConnection is the best WebRTC library I have found, expect the coming v3.0.
